AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -1079,8 +1037,8 @@ public void onBackPressed() {
             super.onBackPressed();
         } else {
             Timber.i("Back key pressed");
-            if (mActionsMenu != null && mActionsMenu.isExpanded()) {
-                mActionsMenu.collapse();
+            if (mFloatingActionMenu.mIsFABOpen) {
```

Comment:
Make the variable private so all of these become compile errors

```suggestion
            if (mFloatingActionMenu.isFABOpen()) {
```

Reply:
o sorry i didn't notice that one , okay i will makes changes accordingly :)